guidelines/monai-application-package.md

```diff
@@ -1,498 +1,588 @@
+  - The value provided must be an absolute path (the first character is `/`).
 
-    - Model names SHALL NOT contain any Unicode whitespace or control characters.
-
-    - Model names SHALL NOT exceed 128 bytes in length.
+  - When not provided, the default path `/opt/holoscan/models/` SHALL be assumed.
 
-  - Models SHOULD provide a file-system path if they're included in the MAP itself (`/etc/monai/pkg.json#models[*].path`).
+- Package Manifest SHOULD list the models used by the application (`/etc/holoscan/pkg.json#models`).
 
-    - Model path SHOULD be relative to the folder containing the model.
+  - Models SHALL be defined by name (`/etc/holoscan/pkg.json#models[*].name`).
 
-    - When the model path is a relative file-system path (first character is not `/`), it is relative to the `/opt/monai/models/` folder.
+    - Model names SHALL NOT contain any Unicode whitespace or control characters.
 
-    - When the model path is an absolute file-system path (first character is `/`), the file-system path is used as-is.
+    - Model names SHALL NOT exceed 128 bytes in length.
 
-  - Models SHOULD be in sub-folders of the `/opt/monai/models/` folder.
+  - Models SHOULD provide a file-system path if they're included in the MAP itself (`/etc/holoscan/pkg.json#models[*].path`).
 
-- Package Manifest SHOULD specify the resources required to execute the application.
+    - When the value is a relative file-system path (the first character is not `/`), it is relative to the model root directory defined in `/etc/holoscan/pkg.json#model-root`.
 
-  This information is used to provision resources when running the application using the MONAI Application Server.
+    - When the value is an absolute file-system path (the first character is `/`), the file-system path is used as-is.
 
-  - CPU requirements SHALL be denoted using decimal count of CPU cores (`/etc/monai/pkg.json#resources.cpu`).
+    - Whe no value is provided, the name is assumed as the name of the directory relative to the model root directory defined in `/etc/holoscan/pkg.json#model-root`.
 
-  - GPU requirements SHALL be denoted using integer count of GPUs (`/etc/monai/pkg.json#resources.gpu`).
+- Package Manifest SHOULD specify the resources required to execute the Application and the fragments for a Multi-Fragment Application.
 
-  - Memory requirements SHALL be denoted using decimal values followed by units (`/etc/monai/pkg.json#resources.memory`).
+  This information is used to provision resources when running the containerized application using a compatible application deployment service.
 
-    - Supported units SHALL be megabytes (`Mi`) and gigabytes (`Gi`).
+- A classic Application or a single Fragment Application SHALL define its resources in the `/etc/holoscan/pkg.json#resource` object.
 
-    - Example: `1.5Gi`, `2048Mi`
+  - The `/etc/holoscan/pkg.json#resource` object CAN be used as a catchall for all fragments.
```

Comment:
Redundant mention of `for all fragments`, as this is the case for classic app or a SINGLE fragment app.

Reply:
This means a multi-fragment application can use the top-level `resources` definition without specifying one for each fragment.

Reply:
This section is for SINGLE fragment app, so it is illogical to refer to multi-fragment apps. Only make sense if the section covers apps of single or multi-fragments all in once.

Reply:
I changed the wording to explicitly say it is for single fragment app, and CAN also be use for multi-fragment app.